lib/group_button.dart

```diff
@@ -58,6 +61,7 @@ class GroupButton extends StatelessWidget {
     Key key,
     @required this.buttons,
     @required this.onSelected,
+    this.selectedButtons,
     this.isRadio = true,
```

Comment:
GroupButton widget can creates radio and check buttons state.
In radio button case we can't create more selectedButtons than 1.

Please create the assert with this case ->
```dart
if(isRadio){
    assert(selectedButtons.length <= 1);
}
```
in **GroupButton** build method

Reply:
I think it be better to check in constructor instead of checking in method called from build. 

```
  const GroupButton({
...
  })  : assert(
            !isRadio || selectedButtons == null || selectedButtons.length <= 1,
            "When isRadio=true, selectedButtons can't contain multiple buttons."),
        super(key: key);
```

It will throw below exception:

```
════════ Exception caught by widgets library ═══════════════════════════════════
The following assertion was thrown building MyHomePage(dirty, state: _MyHomePageState#61460):
When isRadio=true, selectedButtons can't contain muliple buttons.
'package:group_button/group_button.dart':
Failed assertion: line 78 pos 13: '!isRadio || selectedButtons == null || selectedButtons.length <= 1'
```

What do you think?